pkg/service/repair/controller.go

```diff
@@ -3,6 +3,7 @@
 package repair
 
 import (
+	//"fmt"
```

Comment:
?

Reply:
resolved (I removed the last patch from this PR that adds those printouts).
I'll consider sending a followup with a proper logging.